app/server.go

```diff
@@ -35,8 +35,9 @@ func main() {
 	env := &handlers.Env{db}
 
 	r := mux.NewRouter()
-	r.HandleFunc("/karen/v1/users", env.PostUserHandler).Methods("POST")
 	r.HandleFunc("/karen/v1/users/auth", env.PostAuthHandler).Methods("POST")
+	r.HandleFunc("/karen/v1/users", env.PostUserHandler).Methods("POST")
+	r.HandleFunc("/karen/v1/users/", env.PatchUserHandler).Methods("PATCH")
```

Comment:
Just to match what we discussed [here](https://github.com/schramm-famm/karen/pull/8#discussion_r392362332) for the `GET` API, I think the path for `PATCH`ing yourself, should be `/karen/v1/users/self`.

Also, in the `PostUserHandler` method, the `Location` would need to be set to `/karen/v1/users/self` as well.

Reply:
Is that super necessary since you can't `PATCH` anyone's but your own? For example, the `ether` route for getting all of your own conversations is just `GET /ether/v1/conversations`.

Reply:
I think it would be more RESTful to `PATCH` and `GET` (and `DELETE`) on the same route as it's all the same "resource" (yourself).

Like the flow is:
1. Call `POST /karen/v1/users` and that returns a `Location` header of `/karen/v1/users/self`.
2. Call `GET/PATCH/DELETE /karen/v1/users/self` to read/update/delete yourself.

Reply:
Ah okay I get what you mean. Yeah I agree. 👍🏻

Reply:
for the POST /karen/v1/users the location that is currently being returned includes the user ID because there is no "self" yet right?

Reply:
Instead of the user ID, you would just make it `self`, ex: `/karen/v1/users/{user-id}` -> `/karen/v1/users/self`.